core/src/main/java/org/sonarsource/sonarlint/core/container/storage/partialupdate/PartialUpdater.java

```diff
@@ -67,4 +72,14 @@ public void updateFileIssues(String moduleKey, String filePath) {
     }
     issueStore.save(issues);
   }
+
+  public void updateModuleList() {
+    Path moduleListPath = storageManager.getModuleListPath();
+    try {
+      moduleListDownloader.fetchModulesList(moduleListPath);
+    } catch (Exception e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1166)